src/components/ui/MobileDialog.tsx

```diff
@@ -23,14 +23,14 @@ export const DialogContent = React.forwardRef<any, Props>(
           className={clx(fullScreen ? 'dialog-wide' : 'dialog-default')} {...props} ref={forwardedRef}
         >
           <div className='flex items-center px-2'>
+            <DialogPrimitive.Title className='dialog-title'>
```

Comment:
@julianlam after some investigation, I believe the issue is actually that the <PhotoGalleryModal /> component is using the defaults.css css rules, versus what was being tested in https://github.com/OpenBeta/open-tacos/commit/fdb64aa049e37ee0128942d0e21ca3982e1d28ac, that was referencing the global.css file.

I think some evidence to point to this is the new /climb//climbId page when updated to Next13 in https://github.com/OpenBeta/open-tacos/pull/1195, the DialogClose button is correctly showing. This new page would be referencing global.css.

So while your fix works, maybe the fix is instead to alter the css in defaults.css to match global.css so we aren't relying n order, but instead restricting the title from overlapping with the close btuton:

  .dialog-title {
    @apply flex-grow text-center bg-base-100 leading-none;
  }

  .dialog-close-button {
    @apply mx-auto z-auto;
  }
What do you think?

Reply:
Given that the PhotoGalleryModal is only invoked by the climbs page (I don't actually know if this is true), and that it is being refactored, might be this PR is moot and can be closed :smile:

Reply:
For sure, I do think it's still valid to fix though because it definitely is an issue during the transition period of old climb page to new climb page.